publication-commons/src/test/resources/features/CreatorAccessRights.feature

```diff
@@ -0,0 +1,148 @@
+Feature: Creator access rights
+
+  Background:
+    Given that there is a database "PUBLICATIONS" with publications
+    And that DynamoDBPublicationService exists
+    And the DynamoDBPublicationService has a READ method
+    And the READ method requires a user with non empty username
+    And the READ method requires a non empty publication ID
+    And the DynamoDBPublicationService has an UPDATE method
+    And the UPDATE method requires a user with non empty username
+    And the UPDATE method requires a non empty publication ID
+    And the DynamoDBPublicationService has an DELETE method
+    And the DELETE method requires a user with non empty username
+    And the DELETE method requires a non empty publication ID
+    And the DynamoDBPublicationService has a CREATE method
+    And the CREATE method requires a user with non empty username
+    And the CREATE method requires a non empty publication object
+    And the DynamoDBPublicationService has a PUBLISH method
+    And the PUBLISH method requires a user with non empty username
+    And the PUBLISH method requires a non empty publication ID
+
+    And there is an authenticated user
+    And the authenticated user is affiliated with an institution
+    And the user's role is CREATOR
+    And the user does not have any other role
+
+
+  Scenario: uses with CREATOR role can read NOT own published publications
```

Comment:
NOT owned?

Reply:
Yes, everybody should be allowed to read published publications. If an Anonymous user can read published publications, so should a CREATOR as well.